modules/library/metadata/src/test/java/org/geotools/measure/UnitFormatterTest.java

```diff
@@ -0,0 +1,213 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2021, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.measure;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.emptyList;
+import static javax.measure.MetricPrefix.MICRO;
+import static org.junit.Assert.assertEquals;
+
+import java.lang.reflect.Field;
+import java.util.AbstractMap;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import javax.measure.Unit;
+import javax.measure.format.UnitFormat;
+import org.junit.Test;
+import tech.units.indriya.format.SimpleUnitFormat;
+import tech.units.indriya.unit.Units;
+
+public class UnitFormatterTest {
+
+    @Test
+    public void unitsOnlyInOld() throws Exception {
+        SimpleUnitFormat simpleUnitFormat = SimpleUnitFormat.getInstance();
+
+        List<Map.Entry<Unit<?>, String>> unitToName =
+                toSortedList1(getUnitToNameMap(simpleUnitFormat));
+        List<Map.Entry<Unit<?>, String>> unitToSymbol =
+                toSortedList1(formatter.getUnitToSymbolMap());
+
+        // indriya (2.0.2) has an inconsistent mix of μ and µ; this doesn't.
```

Comment:
Please don't leave commented-out code (please check the whole PR, in case there are other bits)

Reply:
Done.